sormas-backend/src/test/resources/checkHistoryTables.sql

```diff
@@ -28,4 +28,50 @@ UNION
 SELECT 'no history table' as remark, t.table_name, null as column_name, null as data_type FROM information_schema."tables" t
 WHERE t.table_schema = 'public' AND t.table_name NOT LIKE '%_history' AND t.table_name NOT IN ('schema_version', 'systemevent') AND t.table_name NOT like 'pg_%'
 AND (SELECT COUNT(t_hist.table_name) FROM information_schema."tables" t_hist WHERE concat(t.table_name,'_history') = t_hist .table_name) = 0
+UNION
+SELECT 'missing delete history trigger' as remark, t.table_name, null as column_name, null as data_type FROM information_schema."tables" t
+WHERE t.table_schema = 'public'
+  AND t.table_name NOT LIKE '%_history'
+  AND t.table_name NOT LIKE '%_%'
+  AND t.table_name NOT IN ('schema_version', 'systemevent')
+  AND t.table_name NOT like 'pg_%'
+  AND NOT exists(SELECT trigger_name
+                 FROM information_schema.triggers
+                 WHERE trigger_name = 'delete_history_trigger'
+                   AND event_object_table = t.table_name)
+UNION
+SELECT 'missing versioning trigger' as remark, t.table_name, null as column_name, 'INSERT' as data_type FROM information_schema."tables" t
+WHERE t.table_schema = 'public'
+  AND t.table_name NOT LIKE '%_history'
+  AND t.table_name NOT IN ('schema_version', 'systemevent')
+  AND t.table_name NOT like 'pg_%'
+  AND NOT exists(SELECT trigger_name
+                 FROM information_schema.triggers
+                 WHERE trigger_name = 'versioning_trigger'
+                   AND event_manipulation = 'INSERT'
+                   AND event_object_table = t.table_name)
+UNION
+SELECT 'missing versioning trigger' as remark, t.table_name, null as column_name, 'UPDATE' as data_type FROM information_schema."tables" t
+WHERE t.table_schema = 'public'
+  AND t.table_name NOT LIKE '%_history'
+  AND t.table_name NOT IN ('schema_version', 'systemevent')
+  AND t.table_name NOT like 'pg_%'
+  AND NOT exists(SELECT trigger_name
+                 FROM information_schema.triggers
+                 WHERE trigger_name = 'versioning_trigger'
+                   AND event_manipulation = 'UPDATE'
+                   AND event_object_table = t.table_name)
+UNION
+SELECT 'missing versioning trigger' as remark, t.table_name, null as column_name, 'DELETE' as data_type FROM information_schema."tables" t
+WHERE t.table_schema = 'public'
+  AND t.table_name ~* '_'
+  AND t.table_name NOT LIKE '%_history'
+  AND t.table_name NOT IN ('schema_version', 'systemevent')
+  AND t.table_name NOT like 'pg_%'
+  AND NOT exists(SELECT trigger_name
+                 FROM information_schema.triggers
+                 WHERE trigger_name = 'versioning_trigger'
+                   AND (event_manipulation = 'DELETE')
+                   AND event_object_table = t.table_name)
 ORDER BY remark, table_name , column_name;
```

Comment:
Let's also include this check:

```suggestion
UNION
SELECT 'invalid versioning trigger' as remark, t.table_name, null as column_name, 'DELETE' as data_type FROM information_schema."tables" t
WHERE t.table_schema = 'public'
  AND NOT (t.table_name ~* '_')
  AND t.table_name NOT LIKE '%_history'
  AND t.table_name NOT IN ('schema_version', 'systemevent')
  AND t.table_name NOT like 'pg_%'
  AND exists(SELECT trigger_name
                 FROM information_schema.triggers
                 WHERE trigger_name = 'versioning_trigger'
                   AND (event_manipulation = 'DELETE')
                   AND event_object_table = t.table_name)
ORDER BY remark, table_name , column_name;
```